mlir/lib/Dialect/Linalg/IR/LinalgInterfaces.cpp

```diff
@@ -70,6 +70,105 @@ bool linalg::isaCopyOpInterface(LinalgOp linalgOp) {
   return llvm::hasSingleElement(linalgOp.getBlock()->getOperations());
 }
 
+//===----------------------------------------------------------------------===//
+// FillOpInterface implementation
+//===----------------------------------------------------------------------===//
+bool linalg::isaFillOpInterface(GenericOp genericOp) {
```

Comment:
It would be helpful instead of a `bool` this returns a `std::optional<Value>` that is the scalar value as well.

Reply:
Fixed. thanks.